### ISSUE TITLE
How to add a fixed custom item at the bottom

### ISSUE THREAD

**Issue by eriknyk** [rob-balfre/svelte-select#435]
How to add a fixed custom item at the bottom, this is useful to add custom actions,
by example if we have a user list, and there is not any suggestion for a given username we can offer create it as an extra action.

Or in the below example, there is a clear action.

<img width="591" alt="Screen Shot 2022-07-14 at 18 40 48" src="https://user-images.githubusercontent.com/205721/179119170-fe064c6c-1761-4fd1-a90a-25155ed3e3ef.png">

Best Regards.

**Comment by rob-balfre** [comment]
@eriknyk it's a totally different layout/design so not without some pretty heavy changes in the List component.  

Or just some CSS and svelte magic!  
https://svelte.dev/repl/9ad3e8e1878349da92fce768d45ee666?version=3.49.0

**Comment by ZerdoX-x** [comment]
@eriknyk Yep. You can create custom `List` and `Item` components and redefine `noOptionsMessage` (or any other functions) so it would render `svelte:component` instead of text. There is almost no documentation about this (https://github.com/rob-balfre/svelte-select/issues/408#issuecomment-1129734988), but you can acheive this buy diving into source code. Good luck!

**Comment by rob-balfre** [comment]
Latest version of beta has some big changes, named slots for example.  Might be handy for this. Thanks
